ts/src/test/static/response/cryptocom.json

```diff
@@ -1809,6 +1809,3043 @@
           }
         ]
       }
+    ],
+    "fetchOrderBook": [
+      {
+        "description": "spot fetch order book",
+        "method": "fetchOrderBook",
+        "input": [
+          "BTC/USDT"
+        ],
+        "httpResponse": {
+          "id": -1,
+          "method": "public/get-book",
+          "code": 0,
+          "result": {
+            "depth": 150,
+            "data": [
+              {
+                "bids": [
```

Comment:
@pcriadoperez static tests are good 😅 but we should try to keep them concise so they're easy to read/update/remove.

If the exchange response is huge, we can manually update  it and keep just 1 or 2 entry (you can see how I have done in methods like fetchPositions that sometimes return every single default position and ends up being thousands of lines

Reply:
hehe good point. I reduced it to a test with a limit of 5

Reply:
@pcriadoperez thanks! 🙂